cms/lib/xblock/tagging/tagged_block_mixin.py

```diff
@@ -0,0 +1,52 @@
+# lint-amnesty, pylint: disable=missing-module-docstring
+from urllib.parse import quote
+
+
+class TaggedBlockMixin:
+    """
+    Mixin containing XML serializing and parsing functionality for tagged blocks
+    """
+
+    def serialize_tag_data(self):
+        """
+        Serialize block's tag data to include in the xml, escaping special characters
+
+        Example tags:
+            LightCast Skills Taxonomy: ["Typing", "Microsoft Office"]
+            Open Canada Skills Taxonomy: ["MS Office", "<some:;,skill/|=>"]
+
+        Example serialized tags:
+            lightcast-skills:Typing,Microsoft Office;open-canada-skills:MS Office,%3Csome%3A%3B%2Cskill%2F%7C%3D%3E
+        """
+        from openedx.core.djangoapps.content_tagging.api import get_content_tags
```

Comment:
Why is this import inside the function? Is there a circular import issue?

Reply:
Since we are importing the `TaggedBlockMixin` in the cms settings and using it in the `XBLOCK_MIXINS`, the `content_tagging` app is not loaded yet, so having this defined outside causes a `django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.` error.